Add profiling info and update gitignore (#31)
mikage-emu/mikage-dev: README.md

* `cmake .. -G Ninja --toolchain conan_toolchain.cmake -DCMAKE_BUILD_TYPE=Release`
* `ninja && sudo ninja install`

NOTE: On Linux, Vulkan libraries are also required and will NOT be built by Conan.

> by Conan.

Reviewer: ```suggestion
```

Processes running in the emulated kernel can be introspected through a simple
debug console available via telnet on port 12347.

## Profiling

Mikage uses [Tracy](https://github.com/wolfpld/tracy) for profiling. To enable it,
make sure to tell Conan to enable profiling in the conanfile.py and rebuild Mikage.

> Get your Tracy server (you can usally install it via your package manager) up and

Reviewer: Tracy doesn't actually ship in that many distributions. Fedora and openSUSE don't have it.

```suggestion
Get your Tracy server (often available from the system package manager) up and
```
